doc/example/model_selection.ipynb

```diff
@@ -375,6 +349,7 @@
    "outputs": [],
    "source": [
     "petab_select_problem.model_space.reset_exclusions()\n",
+    "petab_select_problem.state.reset()\n",
```

Comment:
Could you comment shortly why you're resetting and what exactly?
I can guess what it is, but just for a new user to not be confused or thinks that it's necessary to do always.

Reply:
Good point. It's not a great interface currently because there is an ongoing refactor to move all "state" information to `problem.state`. The state information is currently distributed among several PEtab Select classes, and is unlikely to be resolved before publication.
So to make things simpler for the user here, I changed this code to simply reload the PEtab Select problem, instead of resetting it.